runtime/src/tokio/runtime.rs

```diff
@@ -59,6 +61,26 @@ impl Metrics {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct NetworkConfig {
+    /// Whether or not to enable TCP_NODELAY.
+    ///
+    /// If `None`, the default value is used.
+    tcp_nodelay: Option<bool>,
+
+    /// Read/write timeout for network operations.
+    read_write_timeout: Duration,
```

Comment:
io_uring doesn't have separate read/write timeouts, so I had to combine the read/write timeout like this. The alternative would be to allow the user to pass both read/write timeout and ignore one when passing to io_uring. I thought that'd be misleading. We could change the iouring code to support different timeouts for different ops.